applications/accounting/src/main/groovy/org/apache/ofbiz/accounting/InvoicePerShipmentTests.groovy

```diff
@@ -34,152 +31,155 @@ import org.apache.ofbiz.shipment.packing.PackingSession
+            logInfo('===== >>> Service : updateOrderHeader / invoicePerShipment = N,  Response = ' + serviceResult.responseMessage)
         }
 
         PackingSession packingSession = new PackingSession(dispatcher, userLogin)
-        session.setAttribute("packingSession", packingSession)
+        session.setAttribute('packingSession', packingSession)
         packingSession.setPrimaryOrderId(orderHeader.orderId)
-        packingSession.setPrimaryShipGroupSeqId("00001")
-
-        Map packInput = [:]
-        packInput.orderId = orderHeader.orderId
-        packInput.shipGroupSeqId = "00001"
-        packInput.packingSession = packingSession
-        packInput.nextPackageSeq = 1
-        packInput.userLogin = userLogin
-
-        // Items
-        packInput.selInfo = [_1: "Y"]
-        packInput.pkgInfo = [_1: "1"]
-        packInput.qtyInfo = [_1: "1"]
-        packInput.prdInfo = [_1: productId]
-        packInput.iteInfo = [_1: "00001"]
-        packInput.wgtInfo = [_1: "0"]
-        packInput.numPackagesInfo = [_1: "1"]
-
-        Map serviceResult = dispatcher.runSync("packBulkItems", packInput)
+        packingSession.setPrimaryShipGroupSeqId('00001')
+
+        Map packInput = [
+                orderId: orderHeader.orderId,
+                shipGroupSeqId: '00001',
+                packingSession: packingSession,
+                nextPackageSeq: 1,
+                userLogin: userLogin,
+                selInfo: [_1: 'Y'],
+                pkgInfo: [_1: '1'],
+                qtyInfo: [_1: '1'],
+                prdInfo: [_1: productId],
+                iteInfo: [_1: '00001'],
+                wgtInfo: [_1: '0'],
+                numPackagesInfo: [_1: '1']
+        ]
+
+        Map serviceResult = dispatcher.runSync('packBulkItems', packInput)
         assert ServiceUtil.isSuccess(serviceResult)
-        logInfo("===== >>> Service: packBulkItems, Response = " + serviceResult.responseMessage)
+        logInfo('===== >>> Service: packBulkItems, Response = ' + serviceResult.responseMessage)
 
-        Map completePackInput = dispatcher.getDispatchContext().makeValidContext("completePack", ModelService.IN_PARAM, packInput)
-        serviceResult = dispatcher.runSync("completePack", completePackInput)
+        Map completePackInput = dispatcher.getDispatchContext().makeValidContext('completePack', ModelService.IN_PARAM, packInput)
+        serviceResult = dispatcher.runSync('completePack', completePackInput)
         assert ServiceUtil.isSuccess(serviceResult)
-        logInfo("===== >>> Service: completePack, shipmentId = " + serviceResult.shipmentId)
+        logInfo('===== >>> Service: completePack, shipmentId = ' + serviceResult.shipmentId)
 
         // Step 4
-        List invoices = from("OrderItemBillingAndInvoiceAndItem").where("orderId", orderHeader.orderId).queryList()
+        List invoices = from('OrderItemBillingAndInvoiceAndItem').where('orderId', orderHeader.orderId).queryList()
         return invoices
     }
```

Comment:
I see no reasons to remove the tests below. I can see them ran at https://nightlies.apache.org/ofbiz/trunk/tests-results/plugins/html/